core/src/avm2/globals/flash/utils.rs

```diff
@@ -111,6 +114,32 @@ pub fn clear_timeout<'gc>(
     Ok(Value::Undefined)
 }
 
+/// Implements `flash.utils.escapeMultiByte`
+pub fn escape_multi_byte<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    let s = args
+        .get(0)
+        .unwrap_or(&Value::Undefined)
+        .coerce_to_string(activation)?;
+    let bs = s.as_wstr().to_utf8_lossy();
+    let mut buf = WString::new();
+    for b in bs.as_bytes() {
+        if *b == 0 {
+            break;
+        }
+        if b.is_ascii_alphanumeric() {
+            buf.push_char(*b as char);
+        } else {
+            write!(&mut buf, "%{b:X}")?;
```

Comment:
This needs a leading zero for <16:
```
trace(flash.utils.escapeMultiByte("\u0001")); // %01 in Flash, %1 in this PR
```
I think `{b:02X}` does the trick.

Reply:
Indeed! Fixed and added a test which would have caught this.